pupil-spa/src/app/services/question/question.service.ts

```diff
@@ -14,7 +14,8 @@ export class QuestionService {
   protected config: Config;
 
   constructor(protected storageService: StorageService) {
-    this.currentQuestion = 1;
+    // First question shall be 0 to prevent nullification of the first array item
+    this.currentQuestion = 0;
```

Comment:
‼️ Doesn't work for me.  

I see `this.currentQuestion` as being the question sequence number.  This not the index position of some array.

Reply:
@jon-shipley refactored as discussed